app/models/goldencobra/location.rb

```diff
@@ -23,11 +23,17 @@
 module Goldencobra
   class Location < ActiveRecord::Base
     geocoded_by :complete_location, :latitude  => :lat, :longitude => :lng
-    after_validation :geocode, :unless => :skip_geocoding_once_or_always
+    after_validation :safe_geocode, :unless => skip_geocoding_once_or_always
```

Comment:
Got an error here: `method_missing: undefined local variable or method 'skip_geocoding_once_or_always'`
Missing symbol for method name: `:skip_geocoding_once_or_always`

I will add it and force an update.

Reply:
Ah that must have happened during manually converting the Ruby 1.9 Hash syntax to Hash rockets to have them confirm to the rest of the project. Thanks for catching and fixing.